applications/crossbar/src/modules_v2/cb_phone_numbers_v2.erl

```diff
@@ -593,6 +593,24 @@ get_find_numbers_req(Context) ->
       ,{'reseller_id', ResellerId}
       ]).
 
+-spec account_lookup(ne_binary()|'undefined', ne_binary()|'undefined') -> {ne_binary()|'undefined', ne_binary()}.
+account_lookup('undefined', ReqResellerId) ->
+    case kz_services:is_reseller(ReqResellerId)
+        orelse cb_context:is_superduper_admin(ReqResellerId)
+    of
+        'false' -> {'undefined', kz_services:find_reseller_id('undefined')};
```

Comment:
`kz_services:find_reseller_id('undefined')` this returns {'ok', master} or {'error', _}.
but if `ReqResellerId` =/= 'undefined' and is not `reseller` the request is in `fault` and should not be processed.